lib/linguist/languages.yml

```diff
@@ -1411,9 +1411,14 @@ Groff:
   - .n
   - .rno
   - .roff
-  tm_scope: text.groff
+  - .tmac
+  filenames:
+  - mmn
```

Comment:
Where? Searching for [`.mmn`](https://github.com/search?q=extension%3Ammn+NOT+nothack&type=Codehttps://github.com/search?q=extension%3Ammn+NOT+nothack&type=Code) throws up only a handful of results, and none of them are Roff.

As a matter of fact, I'm not sure WHAT [they are](https://github.com/ponweist/Wannier90/blob/d2b16e4ac155ba8e1572b839b7875bbb40142f99/tests/test1/wannier.mmn).